app/models/Organisation.js

```diff
@@ -81,20 +85,46 @@ const orgSchema = new Schema({
     ]
   },
   adminInfo: {
-    type: Object,
-    required: true,
+    name: {
+      type: String,
+      required: true
+    },
+    githubHandle: {
```

Comment:
@Rupeshiya github, slack and other info regarding admin will be fetched from `Users` collection.
This will add duplicate data.

Reply:
So should I make reference to ```Users``` collections here?